src/scripts/main.js

```diff
@@ -1,5 +1,36 @@
 'use strict';
 
+const spider = document.querySelector('.spider');
+const wall = document.querySelector('.wall');
+
 document.addEventListener('click', e => {
-  // write code here
+  const item = e.target;
+
+  if (item !== wall) {
+    return;
+  }
+
+  let coordX = e.offsetX - spider.clientWidth / 2;
+  let coordY = e.offsetY - spider.clientHeight / 2;
+  const maxCoordX = wall.clientWidth - spider.clientWidth;
+  const maxCoordY = wall.clientHeight - spider.clientHeight;
+
+  if (coordX > maxCoordX) {
+    coordX = maxCoordX;
+  }
+
+  if (coordY > maxCoordY) {
+    coordY = maxCoordY;
+  }
+
+  if (coordX < 0) {
+    coordX = 0;
+  }
+
+  if (coordY < 0) {
```

Comment:
Use the 'Math.max()' function to ensure that 'coordY' is not less than 0, e.g. 'coordY = Math.max(coordY, 0)'.